composer.json

```diff
@@ -1,36 +1,65 @@
 {
-  "name": "aoe/restler-examples",
-  "type": "typo3-cms-extension",
-  "description": "A TYPO3-Extension, which contains some examples, how to use/configure the restler-Framework (PHP REST-framework to create REST-API's, in TYPO3 via the TYPO3-Extension 'restler'",
-  "authors": [
-    {
-      "name": "AOE GmbH",
-      "email": "dev@aoe.com"
-    }
-  ],
-  "require": {
-    "aoe/restler": "^2.0",
-    "typo3/cms-core": ">=7.6.0, <=8.7.99"
-  },
-  "require-dev": {
-    "symfony/finder": "2.5.x-dev",
-    "guzzle/guzzle": "~3.9",
-    "behat/behat": "2.5.*@stable",
-    "bshaffer/oauth2-server-php": "v1.0",
-    "justinrainbow/json-schema": "1.5.*"
-  },
-  "autoload": {
-    "psr-4": {
-      "Aoe\\RestlerExamples\\": "Classes"
-    }
-  },
-  "replace": {
-    "restler_examples": "*"
-  },
-  "keywords": [
-    "TYPO3",
-    "Restler",
-    "REST",
-    "API"
-  ]
+    "name": "aoe/restler-examples",
+    "type": "typo3-cms-extension",
+    "description": "A TYPO3-Extension, which contains some examples, how to use/configure the restler-Framework (PHP REST-framework to create REST-API's, in TYPO3 via the TYPO3-Extension 'restler'",
+    "authors": [
+        {
+            "name": "AOE GmbH",
+            "email": "dev@aoe.com"
+        }
+    ],
+    "require": {
+        "php": "^7.2",
+        "ext-json": "*",
+        "aoe/restler": "dev-TYPO3V10",
```

Comment:
Getting changed to stable version after merge-request for restler extension is completed.